tests/modules/notebooks/test_notebook_stack.py

```diff
@@ -15,4 +15,4 @@ def test_notebook_stack(client, sgm_notebook, group):
         username="alice",
         groups=[group.name],
     )
-    assert response.data.updateStack.targetUri == sgm_notebook.notebookUri
```

Comment:
minor: re-add empty line at the end of file

Reply:
Hm, it has an empty line 🤔